### ISSUE TITLE
Add option on toCsv() function to split QNum into two columns

### ISSUE THREAD

**Issue by gasserandreas** [datagrok-ai/public#350]

**Comment by skalkin** [comment]
Depends on #372

**Comment by skalkin** [comment]
Available in v 0.137

```
let nums = ['>5', '6', '<7'];
let t = DG.DataFrame.fromColumns([
  DG.Column.qnum('qnum', 3).init((i) => DG.Qnum.parse(nums[i]))
]);

let csv = t.toCsv({qualifierAsColumn: true});
grok.shell.addTableView(DG.DataFrame.fromCsv(csv));
```
